From 0c8874266922774ba0833e1aec7d3b935313d2d5 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 16 Apr 2010 16:52:22 +0200
Subject: [PATCH 28/40] test: change testsuite to local tcp rather than udp multicast

multicast needs an ethernet device and correct firewall rules
the loopback interface suites better as we only have 2 machines
---
 test/TEST-20-NFS/test.sh      |    4 ++--
 test/TEST-30-ISCSI/test.sh    |    4 ++--
 test/TEST-50-MULTINIC/test.sh |    4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/test/TEST-20-NFS/test.sh b/test/TEST-20-NFS/test.sh
index 380fe01..01e1d7a 100755
--- a/test/TEST-20-NFS/test.sh
+++ b/test/TEST-20-NFS/test.sh
@@ -12,7 +12,7 @@ run_server() {
 
     $testdir/run-qemu -hda server.ext2 -m 256M -nographic \
 	-net nic,macaddr=52:54:00:12:34:56,model=e1000 \
-	-net socket,mcast=230.0.0.1:1234 \
+	-net socket,listen=127.0.0.1:12345 \
 	-serial udp:127.0.0.1:9999 \
 	-kernel /boot/vmlinuz-$KVERSION \
 	-append "root=/dev/sda rw quiet console=ttyS0,115200n81 selinux=0" \
@@ -44,7 +44,7 @@ client_test() {
 
     $testdir/run-qemu -hda client.img -m 256M -nographic \
   	-net nic,macaddr=$mac,model=e1000 \
-  	-net socket,mcast=230.0.0.1:1234 \
+	-net socket,connect=127.0.0.1:12345 \
   	-kernel /boot/vmlinuz-$KVERSION \
   	-append "$cmdline $DEBUGFAIL rdinitdebug rdinfo quiet rdnetdebug ro console=ttyS0,115200n81 selinux=0" \
   	-initrd initramfs.testing
diff --git a/test/TEST-30-ISCSI/test.sh b/test/TEST-30-ISCSI/test.sh
index bde1e37..ba7465a 100755
--- a/test/TEST-30-ISCSI/test.sh
+++ b/test/TEST-30-ISCSI/test.sh
@@ -11,7 +11,7 @@ run_server() {
 
     $testdir/run-qemu -hda server.ext2 -hdb root.ext2 -m 256M -nographic \
 	-net nic,macaddr=52:54:00:12:34:56,model=e1000 \
-	-net socket,mcast=230.0.0.1:1235 \
+	-net socket,listen=127.0.0.1:12345 \
 	-serial udp:127.0.0.1:9999 \
 	-kernel /boot/vmlinuz-$KVERSION \
 	-append "root=/dev/sda rw quiet console=ttyS0,115200n81 selinux=0" \
@@ -35,7 +35,7 @@ run_client() {
 
     $testdir/run-qemu -hda client.img -m 256M -nographic \
   	-net nic,macaddr=52:54:00:12:34:00,model=e1000 \
-  	-net socket,mcast=230.0.0.1:1235 \
+	-net socket,connect=127.0.0.1:12345 \
   	-kernel /boot/vmlinuz-$KVERSION \
 	-append "root=dhcp rw quiet rdinitdebug rdinfo rdnetdebug console=ttyS0,115200n81 selinux=0 $DEBUGFAIL" \
   	-initrd initramfs.testing
diff --git a/test/TEST-50-MULTINIC/test.sh b/test/TEST-50-MULTINIC/test.sh
index 5d714f2..c3e3eb8 100755
--- a/test/TEST-50-MULTINIC/test.sh
+++ b/test/TEST-50-MULTINIC/test.sh
@@ -12,7 +12,7 @@ run_server() {
 
     $testdir/run-qemu -hda server.ext2 -m 256M -nographic \
 	-net nic,macaddr=52:54:00:12:34:56,model=e1000 \
-	-net socket,mcast=230.0.0.1:1234 \
+	-net socket,listen=127.0.0.1:12345 \
 	-serial udp:127.0.0.1:9999 \
 	-kernel /boot/vmlinuz-$KVERSION \
 	-append "selinux=0 root=/dev/sda rdinitdebug rdinfo rdnetdebug rw quiet console=ttyS0,115200n81" \
@@ -46,7 +46,7 @@ client_test() {
   	-net nic,macaddr=52:54:00:12:34:$mac1,model=e1000 \
   	-net nic,macaddr=52:54:00:12:34:$mac2,model=e1000 \
   	-net nic,macaddr=52:54:00:12:34:$mac3,model=e1000 \
-  	-net socket,mcast=230.0.0.1:1234 \
+	-net socket,connect=127.0.0.1:12345 \
   	-kernel /boot/vmlinuz-$KVERSION \
   	-append "$cmdline $DEBUGFAIL rdinitdebug rdinfo rdnetdebug ro quiet console=ttyS0,115200n81 selinux=0 rdshell rdcopystate" \
   	-initrd initramfs.testing
-- 
1.7.0.1

